core/services.go

```diff
@@ -267,3 +270,23 @@ func (app *Application) OnCollectionUpdated(collection string, record map[string
 		}
 	}
 }
+
+func (app *Application) getSurvey(id string) (*model.Survey, error) {
+	return app.storage.GetSurvey(id)
+}
+
+func (app *Application) createSurvey(user *model.User, survey model.Survey) (*model.Survey, error) {
+	survey.ID = uuid.NewString()
+	survey.CreatorID = user.Claims.Subject
+	survey.DateCreated = time.Now().UTC()
```

Comment:
We should also set the app ID and org ID on the survey to match the user's claims (we should not trust the API caller to provide these values correctly)